File: test/birthday_call.py
#!/usr/bin/python3
import time , sys , os
from twilio.rest import Client
sys.path.insert( 1 , os.path.join( sys.path[0] , '..' ) )
from personal import *
TwilioClient = Client( twilioAccountSSID , twilioAuthToken )
toNumber = "+1" + sys.argv[ 1 ]
# machine_detection="Enable" ,
new_call = TwilioClient.calls.create(  url=twilioBirthdayCallURL , to=toNumber , from_=twilioFromNumber , method="POST" )

answered = False
for i in range( 500 ):
    time.sleep( 1 )
    new_call = new_call.update()
    status = new_call.status
    print( status )
    if status == "in-progress":
    	answered = True

File: test/birthday_call.js
const personal = require( "../personal.js" ).twilio;
const process = require( "process" );
const client = require('twilio')( personal.account_ssid , personal.auth_token );


var result = null;
result = client.calls
  .create({
     machineDetection: 'Enable',
     url: personal.calls.birthday.call_url ,
     to: "+1" + process.argv[ 1 ] ,
     from: personal.from_number
   })
  .then(call => console.log(call.sid))
  .done( console.log( result ) );


setInterval( function() {
	console.log( result );
} , 1000 );